app/javascript/components/Reporting/User.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import {
+  Table,
+  Head,
+  HeaderRow,
+  HeaderCell,
+  Body,
+  Row,
+  Cell
+} from '@zendeskgarden/react-tables';
+import { useTranslation } from 'react-i18next'
+
+const Reporting = ({ users }) => {
+  const { t } = useTranslation()
+  return (
+    <div>
+      <Table>
+        <Head>
+          <HeaderRow>
+            {/* TODO: searchable name */}
+            <HeaderCell>{t('volunteer_portal.admin.tab.reporting.name')}</HeaderCell>
+            <HeaderCell>{t('volunteer_portal.admin.tab.reporting.email')}</HeaderCell>
+            <HeaderCell>{t('volunteer_portal.admin.tab.reporting.office')}</HeaderCell>
+            {/* TODO: sort by hours */}
+            <HeaderCell>{t('volunteer_portal.admin.tab.reporting.hours')}</HeaderCell>
+          </HeaderRow>
+        </Head>
+        <Body>
+          {
+            users.map((user, index) =>
+              <Row key={index}>
+                <Cell>{user.name}</Cell>
+                <Cell>{user.email}</Cell>
+                <Cell>{user.office.name}</Cell>
+                <Cell>{user.hours}</Cell>
+              </Row>
+            )
+          }
+        </Body>
+      </Table>
+      {/* TODO: Pagination */}
+    </div>
+  )
+}
+
+export default Reporting
```

Comment:
I feel like this components folder is being abused. IMO this folder should contain components that are reusable throughout the application. Page specific components should go into a components folder under the page's folder.
e.g.
```
components/
- Callout
- Calenda
- OfficeFilter
- etc.
pages/admin/Reporting/components
- ReportPicker.jsx
- ReportHeader.jsx
```

We should probably have a discussion about this at some point.

Reply:
I agree, I've moved the component local to the reporting directory. It did indeed feel weird reaching out to the components folder.